frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.tsx

```diff
@@ -45,6 +45,7 @@ const CropsInfoComponent: React.FC<InputModuleProps> = ({
   farmDetails,
   updateFarmDetails,
   handleFormState,
+  updateNutrientDetails,
```

Comment:
Why is this getting passed here?

Reply:
There are two ways in going in here. If updateNutrientDetails is required, and CropsList is extending InputModuleProps. I would need to pass in that updateNutrientDetails even if I don't use it on CropsList.
Second way is to making it optional and then testing it for validity.

Either way works. but made it optional now, and tested for validity.